### PR TITLE
Update README-ger.md

### PR DESCRIPTION
**Pull request by swetabehera**

### CODE REVIEW

File: README-ger.md [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
 
 # MS-DOS v1.25 und v2.0 Quellcode
-Diese repo enthält den originalen Quellcode und kompilierte Binärdateien für MS-DOS v1.25 und MS-DOS v2.0.
+Dieses Repo enthält den ursprünglichen Quellcode und die kompilierten Binärdateien für MS-DOS v1.25 und MS-DOS v2.0.
 
-Dies sind die gleichen Datein [wie sie ursprünglich am 25. März 2014 mit dem Computer History Museum geteilt wurden]( http://www.computerhistory.org/atchm/microsoft-ms-dos-early-source-code/) und werden in diesem Repo (wieder) veröffentlicht, damit sie leichter zu finden sind, in externen Texten und Werken referenziert werden und erkundungen und Experimentieren für diejenigen ermöglicht wird, die sich für frühe PC-Betriebssysteme interessieren.
+Dies sind die gleichen Dateien [ursprünglich am 25. März 2014 beim Computer History Museum geteilt]( http://www.computerhistory.org/atchm/microsoft-ms-dos-early-source-code/) und sind (neu) veröffentlicht in diesem repo um sie leichter zu machen zu finden, Referenz-zu im externen Schreiben und Arbeiten, und erlauben erkundung und experimentierung für Interessenten an frühen PC-Betriebssystemen.
```
Comment (vardai) [inline README-ger.md]: Repo, not repo.

File: README-ger.md [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
 
 # Lizenz
-Alle veröffentlichte Dateien innerhalb dieses Repos sind unter der [MIT (OSI) License]( https://en.wikipedia.org/wiki/MIT_License) gemäß der [LICENSE file](https://github.com/Microsoft/MS-DOS/blob/master/LICENSE.md) in diesem Repo gespeichert.
+Alle Dateien in diesem Repo sind 
+veröffentlicht unter der [MIT (OSI) Lizenz]( https://en.wikipedia.org/wiki/MIT_License) gemäß der[LIZENZ datei](https://github.com/Microsoft/MS-DOS/blob/master/LICENSE.md) in der Wurzel dieses Repo gespeichert.
 
 # Beitragen!
-Die Quelldateien in diesem Repo dienen der historischen Referenz und bleiben statisch. Bitte senden Sie keine Pull Requests, in denen Änderungen an den Quelldateien vorgeschlagen werden 😊.  
+Die Quelldateien in diesem Repo dienen der historischen Referenz und wird statisch gehalten, also bitte **nicht senden** Pull Requests suggerieren irgendein Änderungen an den Quelldateien, aber fühl dich frei um dieses Repo zu forkieren und zu experimentieren 😊.
```
Comment (vardai) [inline README-ger.md]: Forkieren?

File: README-ger.md [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
+Alle Dateien in diesem Repo sind 
+veröffentlicht unter der [MIT (OSI) Lizenz]( https://en.wikipedia.org/wiki/MIT_License) gemäß der[LIZENZ datei](https://github.com/Microsoft/MS-DOS/blob/master/LICENSE.md) in der Wurzel dieses Repo gespeichert.
 
 # Beitragen!
-Die Quelldateien in diesem Repo dienen der historischen Referenz und bleiben statisch. Bitte senden Sie keine Pull Requests, in denen Änderungen an den Quelldateien vorgeschlagen werden 😊.  
+Die Quelldateien in diesem Repo dienen der historischen Referenz und wird statisch gehalten, also bitte **nicht senden** Pull Requests suggerieren irgendein Änderungen an den Quelldateien, aber fühl dich frei um dieses Repo zu forkieren und zu experimentieren 😊.   
 
-Wenn Sie jedoch zusätzliche Inhalte ohne Quellcode oder Änderungen an Nicht-Quelldateien (z. B. diese README-Datei) einreichen möchten, senden Sie sie bitte per PR, und wir prüfen diese.
+Wenn Sie jedoch möchten Übermitteln Sie zusätzlichen Nicht-Quellinhalt oder Änderungen zu Nicht-Quelldateien (z.B. diese README), Bitte über PR einreichen, und wir werden es überprüfen und bedenke.
```
Comment (vardai) [inline README-ger.md]: Now Sie again after "fühl dich frei"?

File: README-ger.md [hunk 4]
```diff
@@ -1,16 +1,19 @@
...
 
-Dieses Projekt hat die [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).  Weitere Informationen finden Sie unter [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) oder Kontaktieren Sie [opencode@microsoft.com](mailto:opencode@microsoft.com) mit allen zusätzlichen Fragen oder Anmerkungen.
+Dieses Projekt hat adoptiert die [Microsoft Open Source-Verhaltenskodex (
```
Comment (vardai) [inline README-ger.md]: "Hat adoptiert die"???

File: README-ger.md [hunk 5]
```diff
@@ -1,16 +1,19 @@
...
+auf Englisch)](https://opensource.microsoft.com/codeofconduct/).  Weitere Informationen finden Sie unter die [Verhaltenskodex FAQ (
```
Comment (vardai) [inline README-ger.md]: "unter die"???

**Comment by vardai** [comment]
OMG. This gets worse every time I look at it.

**Comment by nailujx86** [comment]
@bitcrazed This one should be closed due to the bad translation and since other PRs fixing translation mistakes have already been merged.

**Pull request closed without merging**